terraform/environments/xhibit-portal/network-infrastructure.tf

```diff
@@ -249,6 +276,30 @@ resource "aws_security_group_rule" "waf_lb_allow_web_users" {
   ipv6_cidr_blocks  = ["::/0"]
 }
 
+resource "aws_security_group_rule" "egress-to-prtg" {
+  depends_on               = [aws_security_group.prtg_lb]
+  security_group_id        = aws_security_group.prtg_lb.id
+  type                     = "egress"
+  description              = "allow portal server access from import server"
+  from_port                = 443
+  to_port                  = 443
+  protocol                 = "TCP"
```

Comment:
Not needed - we're not accessing the portal servers from the PRTG load balancer

Reply:
remarked out